less/textinput.less

```diff
@@ -85,7 +105,7 @@
         .textinput-select-icon {
             display: none;
         }
-        .correct .textinput-correct-icon {
+        .textinput-correct-icon {
```

Comment:
Is there a reason this selector has been modified? This will break the `_canShowMarking` functionality introduced in https://github.com/adaptlearning/adapt-contrib-textInput/pull/84.